### ISSUE TITLE
Using pip-service for admin lookup during builds creates invalid hierarchcies on geonames records

### ISSUE THREAD

**Issue by orangejulius** [pelias/wof-admin-lookup#220]
When using the pip-service (rather than in-memory WOF polygons in the importer itself) to do admin lookup, Geonames records are being assigned administrative areas at all levels of the hierarchy. So for example, a county will be given an admin locality, which is not correct.

This leads to incorrect labels, such as this case:
http://pelias.github.io/compare/#/v1/search%3Fsources=geonames&layers=localadmin&size=1&text=city%20of%20new%20york
<img width="689" alt="screen shot 2018-08-01 at 5 07 47 pm" src="https://user-images.githubusercontent.com/111716/43530280-75568464-95ad-11e8-9195-ddececc23bc5.png">

Basically, it appears the code in https://github.com/pelias/wof-admin-lookup/blob/master/src/getAdminLayers.js which was specifically designed to handle this, is not being applied.
